Jenkinsfile

```diff
@@ -1,12 +1,19 @@
 //
-// CTC Auth Jenkins Pipeline
+// CTC Open Id Connect Jenkins Pipeline
```

Comment:
I don't know why we bother with this comment, the location of the file tells you what it is! I removed them from the cashproof repo.

Reply:
Will remove